micro_http/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
```

Comment:
Any modifications from upstream?

Reply:
Very few. I'm going to submit those small changes upstream later this week.

Reply:
Cool. Maybe we could reference their in-tree version then rather than having a copy?

Reply:
I submitted the changes upstream (https://github.com/firecracker-microvm/firecracker/pull/1310) and they've been approved. Once they're in, I'll submit a follow-up PR for using the Firecracker crate directly.